coinstacks/bitcoin/ingester/src/topology.ts

```diff
@@ -0,0 +1,63 @@
+import { Connection } from 'amqp-ts'
+import { logger } from '@shapeshiftoss/logger'
+
+const BROKER_URL = process.env.BROKER_URL as string
+const COINSTACK = process.env.COINSTACK as string
+
+if (!BROKER_URL) throw new Error('BROKER_URL env var not set')
+if (!COINSTACK) throw new Error('COINSTACK env var not set')
+
+const connection = new Connection(BROKER_URL)
+const deadLetterExchange = `exchange.${COINSTACK}.deadLetter`
+
+const topology: Connection.Topology = {
```

Comment:
@kaladinlight what do you think of using a COINSTACK env to generate coin-specific message queues? (vs hard-coding)

Reply:
Could potentially be used for whatever base set of common exchanges/queues we deem necessary. I could see the possibility where a specific coinstack might need some extra queues for processing unique data set maybe? Then there is the even bigger discussion of can we replace rabbit with something lighter weight to provide similar functionality. I am in camp copy for now until we actually get more than a couple coins or only blockbook coins. Would be easy enough to abstract later if we want.

Reply:
so hardcode to bitcoin then?